### ISSUE TITLE
Consider changing the character used for binding behaviors

### ISSUE THREAD

**Issue by glen-84** [aurelia/binding#227]
As far as I know, an ampersand, on its own, is not valid inside an HTML attribute. You can see how even GitHub shows it in red (I assume that this is an error style):

```html
<div click.delegate="go() | pipe & debounce"></div>
```
Escaped (no error style):

```html
<div click.delegate="go() | pipe &amp; debounce"></div>
```

The same is true for the Atom editor, and possibly others.

Perhaps it would be best to avoid this by using an alternative character/syntax?

A few random options:

```html
<div click.delegate="go() | pipe ^ debounce"></div>
```

```html
<div click.delegate="go() | pipe ~ debounce"></div>
```

```html
<div click.delegate="go() | pipe [debounce]"></div>
```

**Comment by EisenbergEffect** [comment]
It's definitely valid HTML. For example, consider an `href` attribute of a link that has a url with a query string. Most likely the HTML syntax highlighter isn't correct. Many of the highlighers don't actually implement the spec accurately.

**Comment by glen-84** [comment]
My apologies, it seems that ampersands are legal as long as they are not [ambiguous](https://html.spec.whatwg.org/multipage/syntax.html#syntax-ambiguous-ampersand).

I guess it's unlikely that someone would type a semi-colon after the binding expression, although that probably depends on the decision taken WRT allowing multiple statements within a binding. @jdanyow

Contrived example: `click.delegate="x() &debounce; y()"`.

**Comment by nomack84** [comment]
@glen-84 that looks wrong, why having 2 event handler methods? You are seeing this wrong. 'x()' is the handler, '& debounce' the binding behavior, 'y()' isn't needed.

**Comment by glen-84** [comment]
@nomack84,

I was porting an Angular 1 app the other day that had:

```html
ng-change="updateChart(); updateMatches()"
```

... so I just did the same in Aurelia and it worked. Then I read about binding behaviors, and I wasn't able to make them work like this, so I discussed it with @jdanyow and he said that it's quite possible that multiple statements will not be supported in future. I since switched it to using a single method.
